app/scripts/modules/core/src/pipeline/config/stages/managed/importDeliveryConfigStage.ts

```diff
@@ -21,6 +21,7 @@ if (SETTINGS.feature.managedDelivery) {
     validators: [
       {
         type: 'stageOrTriggerBeforeType',
+        // FIXME: this needs to be generalized so that it accepts a rocket trigger in the netflix deployment
```

Comment:
@anotherchrisberry How's your new validator stuff coming along? I'll need some help with this so that it works with Rocket code triggers...